### PR TITLE
Document Microsoft.Data.Sqlite

### PR DESCRIPTION
**Pull request by bricelam**
Depends on https://github.com/dotnet/samples/pull/1846

cc @mairaw

### CODE REVIEW

**Comment by bricelam** [comment]
cc @ajcvickers

**Comment by mairaw** [comment]
There are a couple of warnings there are not related to the missing samples

**Comment by mairaw** [comment]
Rebuilding now that samples are merged.

File: docs/standard/data/sqlite/async.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: Async Limitations - Microsoft.Data.Sqlite
```
Comment (mairaw) [inline docs/standard/data/sqlite/async.md]: I've noticed you're missing the required SEO metadata called description in these articles.

https://review.docs.microsoft.com/en-us/help/contribute/metadata-attributes?branch=master

Comment (bricelam) [reply docs/standard/data/sqlite/async.md]: Updated.

**Review by mairaw — APPROVED** [review]
I didn't go through this carefully but I'm approving so you can release this content.

**Comment by bricelam** [comment]
Thanks for all your help on finding a home for these!

**Pull request merged**